sim/midas/src/main/cc/bridges/synthesized_prints.h

```diff
@@ -10,6 +10,26 @@
 
 #include "bridge_driver.h"
 
+// Bridge Driver Instantiation Template
+#define INSTANTIATE_PRINTF(FUNC,IDX) \
```

Comment:
I like this. I kind of wish we would have done this for other bridge driver instantiations as well

Reply:
I will implement it for some of the others but it'll be on as-needed basis